apps/gro/controllers/confirm.js

```diff
@@ -1,63 +1,37 @@
 'use strict';
 
-var util = require('util');
-var controllers = require('hof').controllers;
-var BaseController = controllers.base;
-var path = require('path');
-var i18n = require('hof').i18n;
+const controllers = require('hof').controllers;
+const BaseController = controllers.base;
+const path = require('path');
+const i18n = require('hof').i18n;
+const Model = require('../../common/models/email');
 
-var Model = require('../../common/models/email');
+module.exports = class ConfirmController extends BaseController {
 
-var ConfirmController = function ConfirmController() {
-  BaseController.apply(this, arguments);
-};
-
-util.inherits(ConfirmController, BaseController);
-
-
-ConfirmController.prototype.saveValues = function saveValues(req, res, callback) {
-
-  var locali18n = i18n({
-    path: path.resolve(
-      __dirname, '../translations/__lng__/__ns__.json'
-    )
-  });
+  saveValues(req, res, callback) {
+    const locali18n = i18n({
+      path: path.resolve(
+        __dirname, '../translations/__lng__/__ns__.json'
+      )
+    });
 
-  locali18n.on('ready', function prepareEmail() {
+    locali18n.on('ready', () => {
 
-    var subject = locali18n.translate('pages.email.information.subject');
+      const subject = locali18n.translate('pages.email.information.subject');
```

Comment:
@JoeChapman Do you mean passing the translation into the 'ready' method on line 18? If so then I'm not sure it can. I think it can only be translated within the ready method, I think to ensure that (and I quote) the resources have finished loading.